pgcli/packages/pgspecial/main.py

```diff
@@ -18,6 +18,32 @@
 class CommandNotFound(Exception):
     pass
 
+
+@export
+class PGSpecial(object):
+    def __init__(self):
+        self.timing_enabled = True
+
+        self.commands = COMMANDS.copy()
+        self.register(self.show_help, '\\?', '\\?', 'Show Help.',
+                      arg_type=NO_QUERY)
+
+    def register(self, *args, **kwargs):
+        register_special_command(*args, command_dict=self.commands, **kwargs)
+
+    def execute(self, cur, sql):
+        return execute(cur, sql, self.commands)
```

Comment:
Hrm. So you envision a scenario where someone imports pgspecial but doesn't instantiate PGSpecial object and merely calls pgspecial.execute(cur, '\d')? 

I suppose that is possible with this current approach, but do we need to cater to that scenario? 

I think we should strive to prescribe a single entry point for the 'execute' api, rather than giving two execute methods where one of them is hanging off the pgspecial module and the other hanging off the PGSpecial object.